PlayerControls/sources/ContentControlsUIProps.swift

```diff
@@ -80,63 +80,66 @@ extension DefaultControlsViewController {
         var liveIndicationViewIsHidden: Bool
         var liveDotColor: UIColor?
         
+        var animationsEnabled: Bool
+        
         //swiftlint:disable function_body_length
         //swiftlint:disable cyclomatic_complexity
         init(props: Props, controlsViewVisible: Bool) {
-            controlsViewHidden = {
+             let controlsHidden: Bool = {
                 let isPlayerAbsent = props.player == nil
                 let isControlsViewHidden = !controlsViewVisible
                 let isAirPlayInactive = !(props.player?.item.playable?.airplay.isActive ?? false)
                 
                 return isPlayerAbsent || (isControlsViewHidden && isAirPlayInactive)
             }()
+            controlsViewHidden = controlsHidden
             
             loading = props.player?.item.playable?.loading ?? false
             
-            playButtonHidden = props.player?.item.playable?.playbackAction.play == nil
+            playButtonHidden = props.player?.item.playable?.playbackAction.play == nil || controlsHidden
```

Comment:
`|| controlsHidden` is everywhere can we make this more unified and straightforward?

Reply:
We've discussed this with @AlexeyDemedetskiy and it was decided to leave it like that until we find we find a better decision.

The initial idea was to make each item decide if it should be hidden or not in `uiProps` by itself, including the option when all controls should be hidden. Right now, no matter what comes from `props` to `uiProps`, if we get from the controller that controls should be hidden, the entire `controlsView` will be hidden. 

So in `uiProps`, for example, we have `some button` visible, but in fact, it is not visible on the screen, because it is hidden with it's superview.

If there're any propositions, I'd be glad to hear.

Reply:
hmm, I got it. Have no better propositions for now :(

Reply:
@RomanTysiachnik Why not to use `uiProps.controlsViewHidden` where you need?

Reply:
@BogdanBilonog Then you will have the same thing - you will have to check two values:
`uiProps.[element]` and `uiProps.controlsViewHidden` to decide if this element should be hidden or not. Because if any of these variables are `true`, the item should be hidden.

Reply:
Maybe move this code to separate PR?